docs/api.rst

```diff
@@ -36,7 +36,7 @@ token, which you can get in your profile. Use it in the ``Authorization`` header
                    latter provides web browser interface for API.
     :reqheader Accept: the response content type depends on
                        :http:header:`Accept` header
-    :reqheader Authorization: optional token to authenticate
+    :reqheader Authorization: optional token to editauthenticate
```

Comment:
This looks wrong...

Reply:
my bad... thanks for heads up... fixed...